.expeditor/dockerfile

```diff
@@ -0,0 +1,67 @@
+# escape=`
```

Comment:
where do we use this image?

Reply:
I included that for manual testing but will remove it since it creates confusion. I'll post it to the infra github repo

Reply:
removed